### PR TITLE
Create GitHub API tests

### PR DESCRIPTION
**Pull request by KrashLeviathan**
### Part 1 of Issue #79 : Create test GitHub account, and create API tests

- Runs tests on the methods that interface with the GitHub REST API.

### CODE REVIEW

File: test/cli-test.js [hunk 1]
```diff
@@ -71,13 +72,17 @@ function captureStream(stream){
 describe('User Preferences:', () => {
   describe('When the user is asked about completing the prerequisites:', () => {
     let hook;
+    let previousLevel;
 
     beforeEach(() => {
       hook = captureStream(process.stdout);
+      previousLevel = winston.level;
```
Comment (KrashLeviathan) [inline test/cli-test.js]: NOTE: This was a bit of scope creep, but I added it to make the tests run better. They would fail if you had the winston logger set to a level other than `info` since this test captures the console output.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,6 +16,28 @@ npm run lint  # runs the linter
 ```
 
 
+## Running the Tests
+
+There's a small amount of configuration needed to run the tests locally.
+Observe the bottom of the `.gitignore` file, where it ignores the
```
Comment (jackcmeyer) [inline CONTRIBUTING.md]: Can you add a hyperlink to this file?

[`.gitignore`](https://github.com/hammer-io/tyr/blob/master/.gitignore)
